README.md

```diff
@@ -57,6 +60,13 @@ Simplified man pages.More information: https://tldr.sh.
    tldr {{git checkout}}
 ```
 
+On the first run, this programme caches all available tldr pages. 
```

Comment:
Probably have a new section named "How it works" ?

Reply:
I think that "How it works" and "Usage" is more or less the same